webdriver/tests/element_click/file_upload.py

```diff
@@ -0,0 +1,17 @@
+from tests.support.asserts import assert_error, assert_success
+from tests.support.inline import inline
+
+def click(session, element):
+    return session.transport.send("POST", "session/{session_id}/element/{element_id}/click"
+                                  .format(session_id=session.session_id,
+                                          element_id=element.id))
+
+
+# 14.1 Element Click File Upload Element Tests Step 3
+
+def test_file_upload_state(session):
+    session.url = inline("<input type=file value=Federer>")
```

Comment:
`value` is misleading here. `<input type=file>` can’t have a prepopulated value.

Reply:
fixed